src/tests/ftest/dfuse/mu_perms.py

```diff
@@ -301,7 +301,8 @@ def run_test_il(self, il_lib=None):
         env_with_il = env_without_il.copy()
         env_with_il.update({
             'LD_PRELOAD': os.path.join(self.prefix, 'lib64', il_lib),
-            'D_IL_REPORT': -1  # Log all intercepted calls
+            'D_IL_REPORT': -1,  # Log all intercepted calls
+            'D_IL_INTERCEPTION_ON': '1'
```

Comment:
If it's possible for permissions to be reported differently, maybe we shouldn't add it for this test?

Reply:
Setting 'D_IL_INTERCEPTION_ON' as '1' in this PR actually behaves same as the code in current master in this case (interception is on as default). I think this env will not affect the permissions reported.